### PR TITLE
[MIRROR] fixes modsuit icons [MDB IGNORE]

### PR DESCRIPTION
**Pull request by SkyratBot**
Original PR: https://github.com/tgstation/tgstation/pull/71998
--------------------

## About The Pull Request
#71908 this pr breaks modsuit icons before the first time they seal by making the icon state be skin-standard-part
this fixes that by making them use base_icon_state instead of initial, and also puts that in other places to reduce hardcoded stuff

## Why It's Good For The Game
le bug is le fix

## Changelog
:cl:
fix: fixes modsuit icons
/:cl:

### CODE REVIEW

File: code/modules/mod/mod_clothes.dm [hunk 1]
```diff
@@ -2,7 +2,12 @@
 	name = "MOD helmet"
 	desc = "A helmet for a MODsuit."
 	icon = 'icons/obj/clothing/modsuit/mod_clothing.dmi'
+<<<<<<< HEAD
 	icon_state = "helmet"
+=======
+	icon_state = "standard-helmet"
+	base_icon_state = "helmet"
+>>>>>>> e79d3bc1c44 (fixes modsuit icons (#71998))
```
Comment (GoldenAlpharex) [inline code/modules/mod/mod_clothes.dm]:
```suggestion
	icon_state = "standard-helmet"
	base_icon_state = "helmet"
```

File: code/modules/mod/mod_clothes.dm [hunk 2]
```diff
@@ -14,14 +19,19 @@
 	name = "MOD chestplate"
 	desc = "A chestplate for a MODsuit."
 	icon = 'icons/obj/clothing/modsuit/mod_clothing.dmi'
+<<<<<<< HEAD
 	icon_state = "chestplate"
+=======
+	icon_state = "standard-chestplate"
+	base_icon_state = "chestplate"
+>>>>>>> e79d3bc1c44 (fixes modsuit icons (#71998))
```
Comment (GoldenAlpharex) [inline code/modules/mod/mod_clothes.dm]:
```suggestion
	icon_state = "standard-chestplate"
	base_icon_state = "chestplate"
```

File: code/modules/mod/mod_clothes.dm [hunk 3]
```diff
@@ -32,7 +42,12 @@
 	name = "MOD gauntlets"
 	desc = "A pair of gauntlets for a MODsuit."
 	icon = 'icons/obj/clothing/modsuit/mod_clothing.dmi'
+<<<<<<< HEAD
 	icon_state = "gauntlets"
+=======
+	icon_state = "standard-gauntlets"
+	base_icon_state = "gauntlets"
+>>>>>>> e79d3bc1c44 (fixes modsuit icons (#71998))
```
Comment (GoldenAlpharex) [inline code/modules/mod/mod_clothes.dm]:
```suggestion
	icon_state = "standard-gauntlets"
	base_icon_state = "gauntlets"
```

File: code/modules/mod/mod_clothes.dm [hunk 4]
```diff
@@ -44,7 +59,12 @@
 	name = "MOD boots"
 	desc = "A pair of boots for a MODsuit."
 	icon = 'icons/obj/clothing/modsuit/mod_clothing.dmi'
+<<<<<<< HEAD
 	icon_state = "boots"
+=======
+	icon_state = "standard-boots"
+	base_icon_state = "boots"
+>>>>>>> e79d3bc1c44 (fixes modsuit icons (#71998))
```
Comment (GoldenAlpharex) [inline code/modules/mod/mod_clothes.dm]:
```suggestion
	icon_state = "standard-boots"
	base_icon_state = "boots"
```

**Review by GoldenAlpharex — APPROVED** [review]
Hopefully this works now.

**Review by Tastyfish — APPROVED** [review]

**Pull request merged**